Cubical/HITs/Join/Properties.agda

```diff
@@ -474,3 +475,10 @@ leftInv (Iso→joinIso is1 is2) (inl x) i = inl (leftInv is1 x i)
 leftInv (Iso→joinIso is1 is2) (inr x) i = inr (leftInv is2 x i)
 leftInv (Iso→joinIso is1 is2) (push a b i) j =
   push (leftInv is1 a j) (leftInv is2 b j) i
+
+
+join-leftUnit : {A : Type ℓ} → isContr (join (Unit* {ℓ'}) A)
+fst join-leftUnit = inl tt*
+snd join-leftUnit (inl tt*) = refl
+snd join-leftUnit (inr a) = push tt* a
+snd join-leftUnit (push tt* a i) j = push tt* a (i ∧ j)
```

Comment:
The name `leftUnit` is a bit ambiguous---it could also refer to the left unit law `join ⊥ A ≃ A`. How about instead calling this `joinAnnihilL`, following the algebra naming conventions ([Cubical/Algebra/NAMING.md](https://github.com/agda/cubical/blob/master/Cubical/Algebra/NAMING.md))?